common/meta/raft-store/src/config.rs

```diff
@@ -74,7 +74,7 @@ pub struct RaftConfig {
     /// This host should be stored in raft store and be replicated to the raft cluster,
     /// i.e., when calling add_node().
     /// Use `localhost` by default.
-    #[clap(long, env = KVSRV_LISTEN_HOST, default_value = "localhost")]
+    #[clap(long, env = KVSRV_ADVERTISE_HOST, default_value = "localhost")]
```

Comment:
The env name `KVSRV_ADVERTISE_HOST` seems weird, why not named it `RAFT_ADVERTISE_HOST` or `METASRV_ADVERTISE_HOST`?